confAllChambers.py

```diff
@@ -3,7 +3,7 @@
 def launch(args):
   return launchArgs(*args)
 
-def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim):
+def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,dictConfig,cName,ztrim):
     import datetime,os,sys
     import subprocess
     from subprocess import CalledProcessError
```

Comment:
Looks like there is a redundant import here, if you're only using `CalledProcessError` you should not have line 8, if you're using more from subprocess I don't think you need Line 8 & 9.